blocks.md

```diff
@@ -1547,6 +1572,44 @@ Key | Value
 
 ###### [↥ back to top](#list-of-available-blocks)
 
+## Workspaces
+
+Creates a block which shows the workspaces on i3/sway compatible window managers. This is mainly usefull in combination with the [Bindingmode](#bindingmode) block to have a style-able replacement for the workspaces block provided with i3-bar and swaybar. Like in the originals, clicking on a workspace changes focus to that workspace.
+
+Currently, the workspaces use the Idle, Good, Critical and Warning colors to show which workspace has focus und/or the urgent hint set.
+
+#### Examples
+
+Shows workspaces stripping their number. Define your workspaces like
+
+```
+set $WS1    1 
+bindsym $mod+F1  workspace $WS1
+...
+```
+
+```toml
+[[block]]
+block = "workspaces"
+strip_workspace_numbers = true
+```
+
+The workspaces block will use the workspace numbers for ordering but will only show the symbols.
+
+#### Options
+
+Key | Values | Required | Default
+----|--------|----------|--------
+`strip_workspace_numbers` | Whether to remove the workspace number when displaying a workspace. | No | `false`
+`strip_workspace_name` | Whether to remove the workspace name when displaying a workspace. | No | `false`
+
+If neither `strip_workspace_numbers` nor `strip_workspace_name` are set, workspace number and name are shown, e.g. the workspace in the example would show as `1 `. Seting either options removes one part of the display, setting both options results in an empty display for each workspace. If a workspace has only the stripped part, e.g. `1` with `strip_workspace_name`, it will show that part, lest there would be nothing shown.
+
+#### TODO
+Find out what the Move, Reload and Rename events do and how to react to them.
```

Comment:
Still WIP?

Reply:
Well, there are these TODOs. Beside that, I'd think at least one person should confirm it actually works?